app/assets/javascripts/angular/directives/index-body.js.erb

```diff
@@ -21,7 +21,7 @@ app.directive('onFinishRender', ['$timeout', function ($timeout){
             // isRTL: true,
             // isFitWidth: true
           });
-        });
+        }, 1000);
```

Comment:
We need a delay, the render on demand doesn't work... See pic. this happens still when you first load the page... or when you hard refresh (clear cache)
![1000 milsec screen shot 2015-01-29 at 12 07 35 am](https://cloud.githubusercontent.com/assets/8570304/5959375/a17042b0-a808-11e4-9d94-5737d387af3a.png)